.github/workflows/wheels.yml

```diff
@@ -0,0 +1,54 @@
+name: Build wheels
+
+on: [push]
+# workflow_dispatch:{}
+
+env:
+  DEBIAN_FRONTEND: noninteractive
+  FORCE_CUDA: 1
+  # all values:
+  # 3.5;3.7;5.0;5.2;5.3;6.0;6.1;6.2;7.0;7.2;7.5;8.0;8.6
+  # with optional +PTX
+  # https://github.com/pytorch/vision/blob/0610b13ac4af3717f538454a9c6b1f441cb386f3/packaging/pkg_helpers.bash#L63
+  # you need at least cuda 5.0 for some of the stuff compiled here.
+  TORCH_CUDA_ARCH_LIST: "6.0 6.1 7.0 7.5 8.6"
+
+jobs:
+  build_wheels:
+    strategy:
+      matrix:
+        # python: ["3.7", "3.8", "3.9", "3.10"]
+        python: ["3.10"]
+         # according to the available pytorch cuda versions
+        # cuda: ["cu113", "cu116"]
+        cuda: ["cu116"]
+    name: py${{ matrix.python }}-${{ matrix.cuda }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 360
+    container:
+      image: python:${{ matrix.python }}-slim
+    steps:
+      - name: Install dependencies
+        run: apt-get update && apt-get install gpg wget git software-properties-common rename -y
+
+      - name: Install nvcc
+        run: >
+          wget https://developer.download.nvidia.com/compute/cuda/repos/debian11/x86_64/cuda-keyring_1.0-1_all.deb &&
+          dpkg -i cuda-keyring_1.0-1_all.deb &&
+          apt-get update &&
+          apt-get install cuda-nvcc-11-8 cuda-libraries-dev-11-8 -y
+
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          path: "."
+
+      - name: Build wheel
+        run: >
+          python3 -m pip install -r requirements.txt --extra-index-url https://download.pytorch.org/whl/${{ matrix.cuda }} &&
+          python3 setup.py bdist_wheel -d dist
```

Comment:
> using ninja will get the runner killed for hugging resources

This can be fixed by setting `MAX_JOBS=2` for instance, but I'm not even sure this would be enough - as the github runners [only have 7GB of RAM](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources)

Reply:
Yeah, I tried with 2, it still got killed.